js-code/index.js

```diff
@@ -0,0 +1,21 @@
+var request = new XMLHttpRequest();
+
+const apiCallOnButtonClick = () => {
+  request.open("GET", "https://randomuser.me/api/?format=json", true);
```

Comment:
Disconnect your laptop from the internet and try to call the button to see if you would see any error. You should be able to understand that the error is because the host in unreachable, for example.

Reply:
you mean "click" the button instead of "call" ?

this error is what i see when i click the button without having internet